demo/admin/src/products/ProductForm.tsx

```diff
@@ -177,6 +179,17 @@ export function ProductForm({ id }: FormProps) {
                         <TextField required fullWidth name="title" label={<FormattedMessage id="product.title" defaultMessage="Title" />} />
                         <TextField required fullWidth name="slug" label={<FormattedMessage id="product.slug" defaultMessage="Slug" />} />
 
+                        <Field
+                            name="priceRange"
+                            label={<FormattedMessage id="product.priceRange" defaultMessage="Price range" />}
+                            fullWidth
+                            required
```

Comment:
Since it's not required in the entity/input it probably shouldn't be required in the form either. 

_Also, `Products` are used a lot for debugging and having many required fields can be annoying when trying to quickly add a few products to try something out locally 😅_ 

```suggestion
```